python/lib/sift_py/upload/config.py

```diff
@@ -0,0 +1,149 @@
+from typing import Dict, List, Optional
+
+from pydantic import BaseModel, ConfigDict, model_validator
+from pydantic_core import PydanticCustomError
+
+VALID_TIME_FORMATS = [
+    "TIME_FORMAT_ABSOLUTE_RFC3339",
+    "TIME_FORMAT_ABSOLUTE_DATETIME",
+    "TIME_FORMAT_ABSOLUTE_UNIX_SECONDS",
+    "TIME_FORMAT_ABSOLUTE_UNIX_MILLISECONDS",
+    "TIME_FORMAT_ABSOLUTE_UNIX_MICROSECONDS",
+    "TIME_FORMAT_ABSOLUTE_UNIX_NANOSECONDS",
+    "TIME_FORMAT_RELATIVE_NANOSECONDS",
+    "TIME_FORMAT_RELATIVE_MICROSECONDS",
+    "TIME_FORMAT_RELATIVE_MILLISECONDS",
+    "TIME_FORMAT_RELATIVE_SECONDS",
+    "TIME_FORMAT_RELATIVE_MINUTES",
+    "TIME_FORMAT_RELATIVE_HOURS",
+]
+
+
+VALID_DATA_TYPES = [
```

Comment:
We have a `ChannelDataType` enum over in `lib/sift_py/ingestion/channel.py`. Any way we could use that instead of hardcoding these values?

Reply:
Is there any reason there isn't something like that for `TIME_FORMATS` too?

Reply:
Per our other discussion - these aren't generated. I will create the TimeFormat enum and we can always replace it with a protobuf generated one later if we want.